ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -2012,14 +2012,20 @@ private static void GenerateTableDefinitionPartContent(TableDefinitionPart table
                 GeneratePivotTableCacheDefinitionPartContent(pivotTableCacheDefinitionPart, pt, context);
 
                 PivotTablePart pivotTablePart;
+                bool createdNewPart = false;
```

Comment:
Maybe do this way?
```
var createNewPart = String.IsNullOrWhiteSpace(pt.RelId);
if (createNewPart)
{
    ...
```
A bit more laconic.

Reply:
Had to google "laconic". :-P